### DISCUSSION TITLE
Slingbox PRO (SB200-100) recovery status

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/SlingerPlayer#20] [General]
hi all, 

can anyone enlighten me if the 
Slingbox PRO (SB200-100)
can  be brought online and what is the process. 

I bought it from eBay, don't know username and password, don't have any other details except MAC address that I have blocked from accessing the internet 

it has not had any network access since before 9 November 2022

thanks in advance.

**Comment by sp222222** [comment]
flipper - although similiar this sub is about taking the slinger stream and playing it on a mediaplayer called slingerplayer by the same author. you need to post and read here 👍 https://github.com/GerryDazoo/Slinger/discussions

**Comment by flipper2222222** [comment]
thanks sp222222, my mistake,  I will repost, cheers
